pulp_container/app/registry_api.py

```diff
@@ -1001,26 +964,45 @@ def put(self, request, path, pk=None):
                 tag = models.Tag.objects.get(name=tag.name, tagged_manifest=manifest)
                 tag.touch()
 
+            add_content_units = [str(tag.pk), str(manifest.pk)]
+
+            if is_manifest_list:
+                add_content_units.extend(
```

Comment:
you could get rid of if/else and purely have `[str(content.pk) for content in chain(found_blobs, found_manifests)]`

if you include in the image manifest `found_blobs` variable also the config blob.